app/controllers/twilio_controller.rb

```diff
@@ -55,34 +47,43 @@ def connect
     # format. Our Ruby library provides a helper for generating one
     # of these documents
     response = Twilio::TwiML::Response.new do |r|
```

Comment:
This should be a `VoiceResponse`, also the block syntax is no longer supported on the latest RC